src/test/packages/22-git-data/zarf.yaml

```diff
@@ -9,19 +9,17 @@ components:
     required: true
     repos:
       # Do a full Git Repo Mirror
-      - https://github.com/defenseunicorns/zarf-public-test.git
+      - https://github.com/zarf-dev/zarf-public-test.git
       # The following performs a full Git Repo Mirror forcing a fallback to host `git`
       - https://dev.azure.com/defenseunicorns/zarf-public-test/_git/zarf-public-test
-      # Perform a full repo mirror of a simple repository with a single branch - (this causes an "already up to date" error in go-git)
-      - https://github.com/defenseunicorns/golang-tekton-hello-world.git
```

Comment:
I'm guessing this was a bug we once ran into and added this to check? Feels like a weird thing to include in an e2e test. There is nothing tekton specific in the test or relating to this repo. I'm hesitant to add another repo here, as it doesn't look like in the code we are doing any special error handling of an, "already up to date", error and any repo we put here could change

Reply:
Had to change it cause the repo was moved to private